### ISSUE TITLE
Install of 0.6.3 Deb package fails

### ISSUE THREAD

**Issue by raesene** [aquasecurity/kube-bench#964]
**Overview**

Trying to install the latest version of kube-bench using the AMD64 deb package fails with an error about a missing directory.

**How did you try to install kube-bench?**

Downloaded the Deb file 

```
wget https://github.com/aquasecurity/kube-bench/releases/download/v0.6.3/kube-bench_0.6.3_linux_amd64.deb
```

Then installed using `dpkg -i`

```
sudo dpkg -i kube-bench_0.6.3_linux_amd64.deb
Selecting previously unselected package kube-bench.
(Reading database ... 144482 files and directories currently installed.)
Preparing to unpack kube-bench_0.6.3_linux_amd64.deb ...
Unpacking kube-bench (0.6.3) ...
dpkg: error processing archive kube-bench_0.6.3_linux_amd64.deb (--install):
 error creating directory './etc/kube-bench/cfg/eks-1.0': No such file or directory
Errors were encountered while processing:
 kube-bench_0.6.3_linux_amd64.deb
```

**What happened?**

An error around creating directories was received (as shown in the output above)

**What did you expect to happen:**

kube bench would be installed :)

**Environment** 

OS: Ubuntu 20.04.2 LTS

**Comment by sushiMix** [comment]
I have seen the same issue on both 0.6.3 and 0.6.2

**Comment by yoavrotems** [comment]
Fixed in tag v0.6.5
